js/app.js

```diff
@@ -0,0 +1,179 @@
+// COMBINE OBJECTS ARRAYS AND FUNCTIONS
+
+// 1.
+
```

Comment:
the idea was you would include the actual question as a comment -- but you have the number so that's a start